pypush/cli/__init__.py

```diff
@@ -36,8 +41,29 @@ def client(
     """
     Connect to the APNs courier and listen for app notifications on the given topic
     """
-    typer.echo("Running APNs client")
-    raise NotImplementedError("Not implemented yet")
+    logging.getLogger("httpx").setLevel(logging.WARNING)
+    with contextlib.suppress(CancelledError):
+        anyio.run(notifications_async, topic, sandbox)
+
+
+async def notifications_async(topic: str, sandbox: bool):
+    async with apns.create_apns_connection(
+        *await apns.activate(),
```

Comment:
Should we provide a method to save the activation certificate + key (in a single .PEM file?)
Should we provide a flag to save the base token + scoped token? Or should we save them in a file as well?

Reply:
Going to deal with this in the next PR